azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/SetupControlBarView.kt

```diff
@@ -98,12 +99,30 @@ internal class SetupControlBarView : LinearLayout {
     private fun setMicButtonState(audioOperationalStatus: AudioOperationalStatus) {
         when (audioOperationalStatus) {
             AudioOperationalStatus.ON -> {
-                micButton.isSelected = true
+                micButton.setCompoundDrawablesWithIntrinsicBounds(
+                    null,
+                    AppCompatResources.getDrawable(
+                        context,
+                        R.drawable.azure_communication_ui_ic_fluent_mic_on_24_filled_composite_button_enabled
+                    ),
+                    null,
+                    null
+                )
+
                 micButton.text =
                     context.getString(R.string.azure_communication_ui_setup_mic_on)
             }
             AudioOperationalStatus.OFF -> {
-                micButton.isSelected = false
+                micButton.setCompoundDrawablesWithIntrinsicBounds(
```

Comment:
This might be a regression, we should probably be using the isSelected on the left

Reply:
mic and camera button, follow up after merge